scenario2/fetch_experiment_results.py

```diff
@@ -0,0 +1,40 @@
+import os
+import subprocess
```

Comment:
Hmm, I see the same scripts in scenario1 and scenario2 folders. Do you think we can create a `common/` and just parameterize certain fields? Otherwise you'd have to maintain two scripts at the same time.